dipy/reconst/mapmri.py

```diff
@@ -1541,7 +1544,7 @@ def mapmri_isotropic_K_mu_dependent(radial_order, mu, rgrad):
 def binomialfloat(n, k):
     """Custom Binomial function
     """
-    return factorial(n) / (factorial(n - k) * factorial(k))
+    return sfactorial(n) / (sfactorial(n - k) * sfactorial(k))
```

Comment:
Since I seem to be unable to debug the file due to some error probably linked to the dipy/direction/probabilistic_direction_getter.pyx file (may be some Cython-related error), I'm not that sure whether mfactorial would be "enough" here (i.e. integers not arrays).